frontend/ui/elements/screensaver_menu.lua

```diff
@@ -83,8 +93,8 @@ return {
                     genMenuItem(_("1 second"), "screensaver_delay", "1"),
                     genMenuItem(_("3 seconds"), "screensaver_delay", "3"),
                     genMenuItem(_("5 seconds"), "screensaver_delay", "5"),
-                    genMenuItem(_("Until a tap"), "screensaver_delay", "tap"),
-                    genMenuItem(_("Until 'exit sleep screen' gesture"), "screensaver_delay", "gesture"),
+                    genMenuItem(getUntilTapOrKeyPressText(), "screensaver_delay", "tap"),
```

Comment:
Not worth a function 80 lines away, you can inline that:
`genMenuItem(Device:isTouchDevice() and _("Until a tap") or _("Until a key press"), "screensaver_delay", "tap"),`

Reply:
(as I did, but opposite in https://github.com/koreader/koreader/pull/12771)